content/get-haiku/release-notes-r1b2.md

```diff
@@ -0,0 +1,271 @@
+
+### Step 1: Getting the Latest Updates
+
+You need to make sure that you have all the package updates that are available for your current release. You can do this by running the following command in the terminal:
+
+```shell script
+pkgman update
+```
+
+### Step 2: Switch to the new repositories
+After this we will switch the package manager to point to the new repositories for beta 2. This will allow the package manager to find the new packages. Run the following commands in the Terminal.
+
+```shell script
+pkgman drop Haiku; pkgman drop HaikuPorts;
+pkgman add-repo https://eu.hpkg.haiku-os.org/haiku/r1beta2/$(getarch)/current
+pkgman add-repo https://eu.hpkg.haiku-os.org/haikuports/r1beta2/$(getarch)/current
+```
+
+### Step 3: Installing the new packages and reboot
+
+Now it is time to download and install the packages. You can do this by issuing the following command in the Terminal:
+
+```shell script
+pkgman full-sync
+```
+
+After that step has been successful, you should immediately reboot the system. You can do this from the Deskbar, or by running ```shutdown -r``` in the terminal.
+
+## Beta 2 Highlights
+
+This release contains 20 months worth of improvements. Below you find some highlights.
+
+### New Input Preferences
+
+The Mouse, Keyboard, Touchpad and Joystick preferences have been consolidated into one Input Preferences.
+
+<img src="/files/get-haiku/input_preferences.png"/><br>
+<small><i>The consolidated Input Preferences, including support for mice with 5 buttons.</i></small>
+
+Additionally, there is now support for mice with more than three buttons.
+
+### DeskBar improvements
+
+The Deskbar has had various improvements. It now has a mini-mode, where the icons on the shelf expand leftwards. You can activate it by dragging the shelf space by the right handle to the top of the screen.
+
+<img src="/files/get-haiku/deskbar_minimode.png"/><br>
+<small><i>New mini-mode to put the Deskbar at the top of the screen.</i></small>
+
+Furthermore, there are more options for showing larger icons, which is especially useful on screens with a higher DPI.
+
+<img src="/files/get-haiku/deskbar_icon_sizes.png"/><br>
+<small><i>More size option for icons in DeskBar.</i></small>
+
+### Hardware Support: xHCI, hda, intel_extreme and NVMe
+
+Haiku's contributors have improved on xHCI, the host controller for USB3. This should make USB3 devices more stable. It also fixes some issues with booting Haiku from a USB3 storage device.
+
+Likewise, the Intel High Definition Audio (`hda`) driver has received some improvements, increasing the number of chipsets that it supports.
+
+The driver for Intel graphics chipsets (`intel_extreme`) has had various improvements. It supports more chipsets, as well as blacklists chipsets that are known not to work. The advantage to the latter is that a user will no longer have to manually disable the driver in the boot menu when their system uses one of the blacklisted chipsets.
```

Comment:
It's not really a blacklist, the unsupported chips are just removed from the driver list of known hardware. But I don't have a better wording, so...